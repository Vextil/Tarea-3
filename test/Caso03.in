# Pila
Desapilar
Apilar uno 5
Desapilar
Desapilar
Apilar uno 9
Apilar dos 3
Apilar tres 8
Desapilar
Desapilar
Desapilar
Desapilar
Apilar uno 9
Apilar dos 3
Apilar tres 8
Apilar uno 9
Apilar dos 3
Apilar tres 8
Apilar uno 9
Apilar dos 3
Apilar tres 8
Apilar uno 9
Apilar dos 3
Apilar tres 8
Apilar uno 9
Apilar dos 3
Apilar tres 8
Apilar uno 9
Apilar dos 3
Apilar tres 8
Apilar uno 9
Apilar dos 3
Apilar tres 8
Apilar uno 9
Apilar dos 3
Apilar tres 8
Apilar uno 9
Apilar dos 3
Apilar tres 8
Apilar uno 9
Apilar dos 3
Apilar tres 8
Apilar uno 59
Apilar dos 53
Apilar tres 58
Apilar uno 69
Apilar dos 63
Apilar tres 68
Desapilar
Desapilar
Desapilar
Desapilar
Desapilar
Desapilar
Desapilar
Fin
