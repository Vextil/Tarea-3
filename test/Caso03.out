1># Pila
2>La pila estaba vacia. 
3>OK
4>(5,uno)
5>La pila estaba vacia. 
6>OK
7>OK
8>OK
9>(8,tres)
10>(3,dos)
11>(9,uno)
12>La pila estaba vacia. 
13>OK
14>OK
15>OK
16>OK
17>OK
18>OK
19>OK
20>OK
21>OK
22>OK
23>OK
24>OK
25>OK
26>OK
27>OK
28>OK
29>OK
30>OK
31>OK
32>OK
33>OK
34>OK
35>OK
36>OK
37>OK
38>OK
39>OK
40>OK
41>OK
42>OK
43>OK
44>OK
45>OK
46>OK
47>OK
48>OK
49>(68,tres)
50>(63,dos)
51>(69,uno)
52>(58,tres)
53>(53,dos)
54>(59,uno)
55>(8,tres)
56>Fin
